### PR TITLE
Fixed issue with flipcard not appearing after a number of clicks on IE11

### PR DESCRIPTION
**Pull request by eleanor-heath**
Description of issue: Preview in IE11
Click on the Flipcard, make sure to click on it a few times
Expected Results:
Flipcard will remain in view, no matter how many times you click.

### CODE REVIEW

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by ethan-lp — APPROVED** [review]
👀

**Review by ethan-lp — APPROVED** [review]
👀

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by ethan-lp — APPROVED** [review]
👀

File: less/flipcard.less [hunk 1]
```diff
@@ -141,9 +141,18 @@
 
     // Show the back item
     .flipcard-item-back {
+      pointer-events: none;
```
Comment (nilslp) [inline less/flipcard.less]: I think this one can be removed now you have added the code below.

Or set this to `pointer-events: all;` and put `pointer-events: none;` on line 109 and get rid of the code block at the bottom.

**Review by ethan-lp — APPROVED** [review]
👀

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by nilslp — APPROVED** [review]

**Pull request merged**